src/Shell/AclShell.php

```diff
@@ -155,7 +155,7 @@ public function delete()
 
         $identifier = $this->parseIdentifier($this->args[1]);
         $nodeId = $this->_getNodeId($class, $identifier);
-        $entity = $this->Acl->{$class}->newEntity(['id' => $nodeId]);
+        $entity = $this->Acl->{$class}->get($nodeId);
```

Comment:
Seems like markNew not working, entity is still marked as new even when markNew is set to false.
I tracked it down to ORM/Marshaller::one(...)

However, i've not done much work with cake so waiting for comments on this. Is @dakota's suggestion right way to do it or should we use isNew( $value ) on new entity after it has been created.

```php
$entity = $this->Acl->{$class}->newEntity(['id' => $nodeId]);
$entity->isNew(false);
```

Here it fails to pass $options for new entity: ORM/Marshaller/one()

```php
public function one(array $data, array $options = [])
    {
        $options += ['validate' => true];
        $propertyMap = $this->_buildPropertyMap($options);
        $schema = $this->_table->schema();
        $entityClass = $this->_table->entityClass();
// *****
// HERE IT DISCARDS $options COMPLETELY
// *****
        $entity = new $entityClass();
        $entity->source($this->_table->alias());
        if (isset($options['accessibleFields'])) {
            foreach ((array)$options['accessibleFields'] as $key => $value) {
                $entity->accessible($key, $value);
            }
        }
```